doc/classes/EditorPlugin.xml

```diff
@@ -444,7 +444,7 @@
 			<param index="2" name="script" type="Script" />
 			<param index="3" name="icon" type="Texture2D" />
 			<description>
-				Adds a custom type, which will appear in the list of nodes or resources. An icon can be optionally passed.
+				Adds a custom type, which will appear in the list of nodes or resources. An editor error is generated when this function is called without the icon parameter.
```

Comment:
```suggestion
				Adds a custom type, which will appear in the list of nodes or resources.
```
Better to just remove, unless it should say that the icon can be `null`, but this description is unnecessary